src/tactic/refine_list.lean

```diff
@@ -0,0 +1,118 @@
+-- Copyright (c) 2019 Lucas Allen. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Lucas Allen
+
+-- TODO Documentation in tactics.md
```

Comment:
This is done, right?

Reply:
Whoops. That's a stray line, I've removed it. The documentation for tactics.md is done.